packages/jest-runner/src/index.ts

```diff
@@ -38,12 +42,17 @@ namespace TestRunner {
   export type TestWatcher = JestTestWatcher;
   export type TestRunnerContext = JestTestRunnerContext;
   export type TestRunnerOptions = JestTestRunnerOptions;
+  export type TestFileEvent = JestTestFileEvent;
 }
 
 /* eslint-disable-next-line no-redeclare */
 class TestRunner {
   private _globalConfig: Config.GlobalConfig;
   private _context: JestTestRunnerContext;
+  public eventEmitter: Emittery;
```

Comment:
I'd prefer this not to be `public` - outside of this class ideally we'd just have `on`, and not allow others to `emit`.

Reply:
can add a public `on` method to `TestRunner` which just proxies to `this.eventEmitter.on`, and make `eventEmitter` `private`

Reply:
if we make it `private` we don't need the `UNSTABLE` rename 🙂